test/catch2/live_restore/api/test_live_restore_fh_extent_import_export.cpp

```diff
@@ -0,0 +1,263 @@
+    SECTION("Test that we cannot import a string with holes beyond the end of the source file")
+    {
+        WTI_LIVE_RESTORE_FILE_HANDLE *lr_fh;
+        create_file(source_file.c_str(), 1000);
+        // We need to create a file in the destination from the get go otherwise we'll initialize
+        // it with one extent size 1000.
+        create_file(dest_file.c_str(), 0);
+        testutil_check(open_lr_fh(env, dest_file.c_str(), &lr_fh, WT_FS_OPEN_CREATE));
+        // The file ends at 999.
+        REQUIRE(__wt_live_restore_fh_import_extents_from_string(
+                  session, (WT_FILE_HANDLE *)lr_fh, "1000-1") == EINVAL);
+    }
+
+    SECTION("Test that we cannot import a string with a zero len extent")
+    {
+        WTI_LIVE_RESTORE_FILE_HANDLE *lr_fh;
+        create_file(source_file.c_str(), 1000);
+        // We need to create a file in the destination from the get go otherwise we'll initialize
+        // it with one extent size 1000.
+        create_file(dest_file.c_str(), 0);
+        testutil_check(open_lr_fh(env, dest_file.c_str(), &lr_fh, WT_FS_OPEN_CREATE));
+        REQUIRE(__wt_live_restore_fh_import_extents_from_string(
+                  session, (WT_FILE_HANDLE *)lr_fh, "0-0") == EINVAL);
+    }
+
+    SECTION("Invalid shape string test #1")
+    {
+        WTI_LIVE_RESTORE_FILE_HANDLE *lr_fh;
+        create_file(source_file.c_str(), 1000);
+        // We need to create a file in the destination from the get go otherwise we'll initialize
+        // it with one extent size 1000.
+        create_file(dest_file.c_str(), 0);
+        testutil_check(open_lr_fh(env, dest_file.c_str(), &lr_fh, WT_FS_OPEN_CREATE));
+        REQUIRE(__wt_live_restore_fh_import_extents_from_string(
+                  session, (WT_FILE_HANDLE *)lr_fh, "-") == EINVAL);
+    }
+
+    SECTION("Invalid shape string test #2")
+    {
+        WTI_LIVE_RESTORE_FILE_HANDLE *lr_fh;
+        create_file(source_file.c_str(), 1000);
+        // We need to create a file in the destination from the get go otherwise we'll initialize
+        // it with one extent size 1000.
+        create_file(dest_file.c_str(), 0);
+        testutil_check(open_lr_fh(env, dest_file.c_str(), &lr_fh, WT_FS_OPEN_CREATE));
+        REQUIRE(__wt_live_restore_fh_import_extents_from_string(
+                  session, (WT_FILE_HANDLE *)lr_fh, "-1") == EINVAL);
+    }
+
+    SECTION("Invalid shape string test #3")
+    {
+        WTI_LIVE_RESTORE_FILE_HANDLE *lr_fh;
+        create_file(source_file.c_str(), 1000);
+        // We need to create a file in the destination from the get go otherwise we'll initialize
+        // it with one extent size 1000.
+        create_file(dest_file.c_str(), 0);
+        testutil_check(open_lr_fh(env, dest_file.c_str(), &lr_fh, WT_FS_OPEN_CREATE));
+        REQUIRE(__wt_live_restore_fh_import_extents_from_string(
+                  session, (WT_FILE_HANDLE *)lr_fh, "string1") == EINVAL);
+    }
+
+    SECTION("Invalid shape string test #3")
```

Comment:
If the only thing that changes is the invalid string we could merge these into a single section with 
```
REQUIRE(a)
REQUIRE(b)
REQUIRE(c)
```

if we keep them separate this should become `string test #4`

Reply:
[4b58531](https://github.com/wiredtiger/wiredtiger/pull/11505/commits/4b585318288245f8b3102f83039d37f9d7b0fea1)